packages/grid/_modules_/grid/hooks/features/infiniteLoader/useGridInfiniteLoader.ts

```diff
@@ -70,14 +69,14 @@ export const useGridInfiniteLoader = (
       ) {
         const rowScrollEndParam: GridRowScrollEndParams = {
           visibleColumns,
-          viewportPageSize: containerSizes.viewportPageSize,
-          virtualRowsCount: containerSizes.virtualRowsCount,
+          viewportPageSize: dimensions.rowsInViewportCount,
```

Comment:
Maybe the naming are to be improved.
In the previous implementation I think their were inconsistencies in what "virtual" meant.

In this PR, `rowsInViewportCount` is the amount of rows that are displayed on the screen and `virtualRowsCount` the amount of rows in the current page.
You can have `virtualRowsCount: 100000` and `rowsInViewportCount: 12`.


`rowsInViewportCount` is used for the `rowsScrollEnd` event and is the equivalent of the old `viewportPageSize`.
I tried to avoid any breaking change to be able to merge this one after v5

```ts
export interface GridRowScrollEndParams {
  /**
   * The number of rows that fit in the viewport.
   */
  viewportPageSize: number;
  /**
   * The number of rows allocated for the rendered zone.
   */
  virtualRowsCount: number;
  /**
   * The grid visible columns.
   */
  visibleColumns: GridColumns;
}
```

Reply:
I named it `rowsInViewportCount` and not `viewportPageSize` because it can be less than what would fit in the viewport.
If you have enough space on your scree to display 20 rows but you give 3, `rowsInViewportCount` will be equal to 3 not 20.

Reply:
> I tried to avoid any breaking change to be able to merge this one after v5

My idea would be to calculate this value in `useGridInfiniteLoader`. If it's not used in some other place we keep its calculation where it's used. If other feature asks for this value, then move it to the dimensions object. I want to avoid leaking this values that were created for a single feature.

Reply:
It is also used in the keyboard navigation for the `Page Up` / `Page Down`